admin_guide/garbage_collection.adoc

```diff
@@ -32,14 +38,14 @@ The policy for container garbage collection is based on three node settings:
 
 |`*minimum-container-ttl-duration*`
 |The minimum age that a container is eligible for garbage collection. The
-default is *1m* (one minute). Use *0* for no limit. Values for this setting can be
+default is *0*. Use *0* for no limit. Values for this setting can be
 specified using unit suffixes such as *h* for hour, *m* for minutes, *s* for seconds.
 
 |`*maximum-dead-containers-per-container*`
-|The number of instances to retain per pod container. The default is *2*.
+|The number of instances to retain per pod container. The default is *1*.
 
 |`*maximum-dead-containers*`
-|The maximum number of total dead containers in the node. The default is *240*.
+|The maximum number of total dead containers in the node. The default is *-1*.
```

Comment:
Might indicated that -1 means unlimited

Also all 3 of these flags are deprecated.

Reply:
What are the new flags?

Reply:
https://kubernetes.io/docs/tasks/administer-cluster/out-of-resource/#deprecation-of-existing-feature-flags-to-reclaim-disk

Reply:
@sjenning Hmm so if these are all deprecated and have no direct replacements according to that link you shared, I'm not sure what to do. It was indicated that the docs mostly need to have these values updated, but I'm not sure when the flags became deprecated and what versions of our docs they need to be removed from.

Reply:
@ahardin-rh they are deprecated at least as far back as 3.6, possibly 3.5 as well.  @derekwaynecarr do you know when eviction deprecated these flags?

Reply:
@derekwaynecarr Any guidance here? I want to be sure that when we update the docs across versions, we're not including any outdated information, including deprecated flags. Thanks!

Reply:
@sjenning is there anyone else who can answer this other than @derekwaynecarr ?

Reply:
@sjenning @derekwaynecarr PTAL. I'm currently blocked and not sure what is needed here.

Reply:
the flags were deprecated in kube 1.4, but still remain through kube 1.10.

we should continue to document them as I do not see them being removed for the next 6 mos.